client/components/emails/email-product-price/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { currentUserHasFlag, getCurrentUser } from 'calypso/state/current-user/selectors';
+import { DOMAINS_WITH_PLANS_ONLY } from 'calypso/state/current-user/constants';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class EmailProductPrice extends React.Component {
+	static propTypes = {
+		isLoading: PropTypes.bool,
+		price: PropTypes.string,
+		salePrice: PropTypes.string,
+	};
+
+	static defaultProps = {
+		isMappingProduct: false,
+	};
+
+	renderContentText() {
+		const message = this.props.translate( 'Free for first three months' );
+
+		return <div className="email-product-price__free-text">{ message }</div>;
+	}
+
+	renderContentPrice() {
+		const threeMonths = 86400 * 90 * 10000;
+		const startDate = Date.now() + threeMonths;
```

Comment:
It will likely be much better to use our `localizedMoment` wrapper around `moment` here, for both the addition and the localization/formatting.
```suggestion
		const startDate = moment().add( 3, 'months' ).format( 'L' );
```